README.md

```diff
@@ -25,21 +27,29 @@ cmake --build build
 ctest --test-dir build
 ```
 
-### `OPENASSETIOTEST_ENABLE_SUBPROJECT=OFF` (default)
+### Dependency Modes
 
-When `OPENASSETIOTEST_ENABLE_SUBPROJECT=OFF` (which is the default if
-unspecified), OpenAssetIO will be discovered as a CMake package
-(`find_package(OpenAssetIO)`) and so OpenAssetIO must be pre-built and
-installed into one of [find_package](https://cmake.org/cmake/help/v3.21/command/find_package.html)'s
+This project has two main dependencies, `OpenAssetIO` and
+`OpenAssetIO-MediaCreation`. They can be discovered in two ways.
+
+#### CMake Packages
+
+When either `OPENASSETIOTEST_ENABLE_OPENASSETIO_SUBPROJECT` or
+`OPENASSETIOTEST_ENABLE_MEDIACREATION_SUBPROJECT` are set to `OFF`
+(which is the default if unspecified), the package will be discovered as
+a CMake package (`find_package(...)`) and so the package must be
+pre-built and installed into one of
+[find_package](https://cmake.org/cmake/help/v3.21/command/find_package.html)'s
 configured search paths.
 
-### `OPENASSETIOTEST_ENABLE_SUBPROJECT=ON`
+#### Subprojects
 
-When `OPENASSETIOTEST_ENABLE_SUBPROJECT=ON`, the `OpenAssetIO` git
-submodule under this repository must be populated. OpenAssetIO will then
-be built as a subproject (`add_subdirectory(OpenAssetIO)`) and so all
-its dependencies must be discoverable by CMake. See the OpenAssetIO
-repository for more information.
+When either `OPENASSETIOTEST_ENABLE_OPENASSETIO_SUBPROJECT` or
+`OPENASSETIOTEST_ENABLE_MEDIACREATION_SUBPROJECT`are set to `ON`, the
+package git submodule under this repository must be populated. The
+package will then be built as a subproject (`add_subdirectory(...)`) and
+so all its dependencies must be discoverable by CMake. See the
+OpenAssetIO/OpenAssetIO-MediaCreation repositories for more information.
```

Comment:
[consider][minor] "OpenAssetIO/OpenAssetIO-MediaCreation" looked to me at first glance as `<organisation>/<respository>` rather than two repos.
```suggestion
OpenAssetIO and OpenAssetIO-MediaCreation repositories for more information.
```

Reply:
aye fair play.